apex/amp/_amp_state.py

```diff
@@ -4,14 +4,7 @@
 # http://effbot.org/pyfaq/how-do-i-share-global-variables-across-modules.htm
 import os
 import torch
-
-TORCH_MAJOR = int(torch.__version__.split('.')[0])
-TORCH_MINOR = int(torch.__version__.split('.')[1])
-
-if TORCH_MAJOR == 0:
-    import collections.abc as container_abcs
-else:
-    from torch._six import container_abcs
+import collections.abc as container_abcs
```

Comment:
I'd rather keep existing import for Pytorch 1.7 and earlier:
```python
TORCH_MAJOR = int(torch.__version__.split('.')[0])
TORCH_MINOR = int(torch.__version__.split('.')[1])

if TORCH_MAJOR == 1 and TORCH_MINOR < 8:
    from torch._six import container_abcs
else:
    import collections.abc as container_abcs
```

Reply:
Thanks for your suggestion. I've updated it.